frontend/src/scenes/insights/InsightTabs/InsightDisplayConfig.tsx

```diff
@@ -23,7 +23,7 @@ export function InsightDisplayConfig({
     horizontalUI,
     ...props
 }: InsightDisplayConfigProps & { horizontalUI: boolean }): JSX.Element {
-    return horizontalUI ? (
+    return horizontalUI && props.activeView !== ViewType.FUNNELS ? (
```

Comment:
I think this extra check is not needed as you already check for the funnels view when passing down `horizontalUI`